### PR TITLE
fix for invalid date displayed on cluster page

### PR DESCRIPTION
**Pull request by niravparikh05**
### What does this PR change?

- Fixes Issue #106 

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- No

### Checklist

I confirm, that I have...

- [x] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [x] Formatted the code using `npm run format` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [x] Updated `CHANGELOG.md`

### CODE REVIEW

File: src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js [hunk 1]
```diff
@@ -28,6 +28,33 @@ function isClusterAvailable(cluster) {
   return ready;
 }
 
+function getClusterLastCheckedIn(cluster) {
+  let ready = false;
+  if (
+    cluster?.spec.clusterData &&
+    cluster?.spec.clusterData.cluster_status &&
+    cluster?.spec.clusterData.cluster_status.conditions &&
+    cluster?.spec.clusterData.cluster_status.conditions.length > 0
```
Comment (meain) [inline src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js:37]: I guess we can do `cluster?.spec?.clusterData?.cluster_status?.conditions?.length` here instead.

File: src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js [hunk 2]
```diff
@@ -28,6 +28,33 @@ function isClusterAvailable(cluster) {
...
+  ) {
+    for (
+      let index = 0;
+      index < cluster?.spec.clusterData?.cluster_status?.conditions?.length;
```
Comment (meain) [inline src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js:41]: We can probably access without  `?.` as we have already validated that it exists above.

File: src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js [hunk 3]
```diff
@@ -61,8 +89,15 @@ function ClusterStatusInfo({
                   }}
                 >
                   Last check in&nbsp;
-                  {cluster.metadata.modifiedAt ? (
-                    <span>{Moment(cluster.metadata.modifiedAt).fromNow()}</span>
+                  {lastCheckedIn != null ? (
+                    <span>
```
Comment (meain) [inline src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js:93]: We have [DateFormat](https://github.com/RafayLabs/rcloud-console-ui/blob/main/src/components/DateFormat.js) if you want to switch to that.

Comment (niravparikh05) [reply src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js:93]: what we display here is not the DateTime similar to project creation. It's like this "Last check-in  6 days ago" and for that we need to use Moment.

File: src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js (hunk 1 above)
Comment (niravparikh05) [reply src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js]: makes sense, changed this.

File: src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js (hunk 2 above)
Comment (niravparikh05) [reply src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js]: yeah, removed that and added this before for loop.

**Review by meain — APPROVED** [review]
LGTM

**Pull request merged**